[2025-03-26T02:48:45.338+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-03-26T02:48:45.386+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_to_s3_to_clickhouse.upload_to_s3 manual__2025-03-26T02:45:11.132497+00:00 [queued]>
[2025-03-26T02:48:45.400+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_to_s3_to_clickhouse.upload_to_s3 manual__2025-03-26T02:45:11.132497+00:00 [queued]>
[2025-03-26T02:48:45.400+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-03-26T02:48:45.423+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): upload_to_s3> on 2025-03-26 02:45:11.132497+00:00
[2025-03-26T02:48:45.435+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'api_to_s3_to_clickhouse', 'upload_to_s3', 'manual__2025-03-26T02:45:11.132497+00:00', '--job-id', '52', '--raw', '--subdir', 'DAGS_FOLDER/extract.py', '--cfg-path', '/tmp/tmp0n45echo']
[2025-03-26T02:48:45.441+0000] {standard_task_runner.py:91} INFO - Job 52: Subtask upload_to_s3
[2025-03-26T02:48:45.445+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=974) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-26T02:48:45.446+0000] {standard_task_runner.py:63} INFO - Started process 989 to run task
[2025-03-26T02:48:45.515+0000] {task_command.py:426} INFO - Running <TaskInstance: api_to_s3_to_clickhouse.upload_to_s3 manual__2025-03-26T02:45:11.132497+00:00 [running]> on host 7ea2041f8442
[2025-03-26T02:48:45.644+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='api_to_s3_to_clickhouse' AIRFLOW_CTX_TASK_ID='upload_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2025-03-26T02:45:11.132497+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-26T02:45:11.132497+00:00'
[2025-03-26T02:48:45.645+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-03-26T02:48:45.694+0000] {credentials.py:1147} INFO - Found credentials in environment variables.
[2025-03-26T02:48:51.437+0000] {logging_mixin.py:188} INFO - âœ… Uploaded to S3: s3://my-tf-data-bucket-hackathon/raw/all_college_data.json
[2025-03-26T02:48:51.439+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-03-26T02:48:51.439+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-03-26T02:48:51.453+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=api_to_s3_to_clickhouse, task_id=upload_to_s3, run_id=manual__2025-03-26T02:45:11.132497+00:00, execution_date=20250326T024511, start_date=20250326T024845, end_date=20250326T024851
[2025-03-26T02:48:51.509+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-03-26T02:48:51.548+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-26T02:48:51.553+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
